### ISSUE TITLE
Toggle default category triggering global update 

### ISSUE THREAD

**Issue by RedIODev** [tachiyomiorg/tachiyomi#9646]
### Describe your suggested feature

It should be a simple toggle that decides rather updating the default category updates all other categories with it. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by scb261** [comment]
You already can manually trigger global update - tap on the three dots at the top and choose `Global update`

**Comment by RedIODev** [comment]
Yes but the default category updates all others with it (global update). The idea is a toggle to disable that difference between default and other categories so that default updates individually from the user created categories.

**Comment by scb261** [comment]
I just tested with both r5863 and 0.14.6, and both of them updated only the default category without triggering global update
